ui/src/app/pages/homepage/registration-dashboard.tsx

```diff
@@ -102,34 +102,35 @@ export const getRegistrationTasks = () => serverConfigStore.getValue() ? ([
       return profile.twoFactorAuthCompletionTime || profile.twoFactorAuthBypassTime;
     },
     onClick: redirectToGoogleSecurity
+  }, {
+    key: 'eraCommons',
+    completionPropsKey: 'eraCommonsLinked',
+    title: 'Connect Your eRA Commons Account',
+    description: 'Connect your account to the eRA Commons account. The application instructions will guide you through this.',
+    buttonText: 'Login',
```

Comment:
I know this matches the mocks, but I wonder if "Connect" would be a better choice for this button.